typescript/agentkit/README.md

```diff
@@ -591,6 +594,39 @@ const walletData = await walletProvider.exportWallet();
 ```
 
 
+SVM:
+- [SolanaKeypairWalletProvider](./src/wallet-providers//solanaKeypairWalletProvider.ts)
+
+### SolanaKeypairWalletProvider
+
+The `SolanaKeypairWalletProvider` is a wallet provider that uses the API [Solana web3.js](https://solana-labs.github.io/solana-web3.js/).
+
+#### Solana Network Configuration
+
+The `SolanaKeypairWalletProvider` can be configured to use a specific network by passing the `networkId` parameter to the `fromNetwork` method. The `networkId` is the ID of the Solana network you want to use. Valid values are `solana-mainnet`, `solana-devnet` and `solana-testnet`.
```

Comment:
please call out that this configuration will result in default RPCs and list which RPCs are used